packages-hybrid/hybrid-mobile-actions/package.json

```diff
@@ -9,11 +9,10 @@
     "build": "npm run build:tsc",
     "prebuild:tsc": "..\"/../node_modules/.bin/rimraf\" dist/tsc",
     "build:tsc": "..\"/../node_modules/.bin/tsc\" --project src/tsconfig.json",
-    "postbuild": "..\"/../node_modules/.bin/ts-node\" --project ../../scripts/modules/tsconfig.json ../../scripts/modules/copyActionsToTestProject.ts",
```

Comment:
A change that is also nice to mention in PR description ;)

Reply:
Done

Reply:
🤔
 `Execute release instead of build in the e2e test phase (ignoring utils-react-widgets)`

Reply:
Done